ray-operator/test/e2e/support.go

```diff
@@ -175,3 +178,54 @@ func jobSubmitterPodTemplateApplyConfiguration() *corev1ac.PodTemplateSpecApplyC
 						corev1.ResourceMemory: resource.MustParse("500Mi"),
 					}))))
 }
+
+func IsEnvVarExisted(env string, pod *corev1.Pod) bool {
+	for _, envVar := range pod.Spec.Containers[utils.RayContainerIndex].Env {
+		if envVar.Name == env {
+			return true
+		}
+	}
+	return false
+}
+
+func deployRedisWithoutPassword(test Test, g *WithT, namespace string) {
```

Comment:
Avoid this kind of pass-through function. Instead, users should call `deployRedis(test, g, namespace, "")` directly. This is one of the recommendations from the book "A Philosophy of Software Design".